app/models/offence.rb

```diff
@@ -35,12 +35,12 @@ def to_builder
       offence.modeOfTrial modeOfTrial
       offence.wording wording
       offence.wordingWelsh wordingWelsh
-      offence.startDate startDate.to_date
-      offence.endDate endDate.to_date
-      offence.arrestDate arrestDate.to_date
-      offence.chargeDate chargeDate.to_date
-      offence.laidDate laidDate.to_date
-      offence.dateOfInformation dateOfInformation.to_date
+      offence.startDate startDate&.to_date
```

Comment:
startDate being `required` I would omit this one.

Reply:
done